### PR TITLE
Remove installers and tools pages, rename machines to containers

### PR DESCRIPTION
**Pull request by ashumilova**
Signed-off-by: Anna Shumilova <ashumilo@redhat.com>

### What does this PR do?
- Removed Installers and Tools pages from workspace details
- Renamed machines to containers
- Fixed selenium tests

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/12909

### CODE REVIEW

**Comment by ashumilova** [comment]
ci-test

**Comment by ashumilova** [comment]
ci-test

**Comment by ashumilova** [comment]
@eclipse/eclipse-che-qa can you, please, verify the test report and changes in tests codebase?

**Comment by SkorikSergey** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1702//Selenium_20tests_20report/) doesn't show any regression against this Pull Request.

**Comment by ashumilova** [comment]
ci-test

File: dashboard/src/app/workspaces/workspace-details/environments/environments.html [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
   <!-- Machines -->
```
Comment (olexii4) [inline dashboard/src/app/workspaces/workspace-details/environments/environments.html]: ```<!-- Containers -->```

**Review by olexii4 — APPROVED** [review]

**Comment by ashumilova** [comment]
ci-test

**Comment by ashumilova** [comment]
ci-test

**Comment by ashumilova** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Pull request merged**